presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/DockerContainer.java

```diff
@@ -75,7 +77,13 @@
             .with(Executors.newCachedThreadPool(daemonThreadsNamed("docker-container-%d")));
 
     private final String logicalName;
-    private final Stopwatch startupTime = Stopwatch.createUnstarted();
+
+    // start is retried, we are recording the last attempt only
+    @GuardedBy("this")
+    private OptionalLong lastStartUpCommenceTimeNanos = OptionalLong.empty();
```

Comment:
what is `Commence`? Use stupid English.

Reply:
Also let names to match the methods

Reply:
would `startUpStartUpTime" be better? that's why i used this word.

Reply:
startUpBeginTime and startUpEndTime?

Reply:
i like my names more. Can it be the day when we learn a new word? 
i can still change if you want to.

Reply:
keep them